src/genegraph/transform/gene_validity_refactor.clj

```diff
@@ -118,8 +117,8 @@
             ;; "Candidate gene sequencing" "gcixform:CandidateGeneSequencing"
             ;; "Exome/genome or all genes sequenced in linkage region" "gcixform:ExomeSequencing"
 
-            "Candidate gene sequencing" "SEPIO:0004543"
-            "Exome/genome or all genes sequenced in linkage region" "SEPIO:0004541"
+            "Candidate gene sequencing" "http://purl.obolibrary.org/obo/SEPIO_0004543"
```

Comment:
I believe these should be able to use the `SEPIO:` prefix if the definition of the `SEPIO` term in the context above explicitly sets `@prefix: true`.

```
"SEPIO": {"@id": "http://purl.obolibrary.org/obo/SEPIO_", "@prefix": true}
```

See this thread: https://github.com/w3c/json-ld-syntax/issues/329#issuecomment-592090824